test/integration/action/action-integration.spec.js

```diff
@@ -105,10 +106,12 @@ describe('Action Integration Tests', function() {
       isDev,
       logger,
       btcdSettingsDir: BTCD_SETTINGS_DIR,
+      miningAddress: BTCD_MINING_ADDRESS,
     };
     btcdProcess = await startBtcdProcess(btcdArgs);
     await nap(NAP_TIME);
     await retry(() => isPortOpen(BTCD_PORT));
+    await mineBlocks({ blocks: 400, logger });
```

Comment:
Curious why we need to mine block before even starting the lnd processes?

Reply:
There's been a change to neutrino such that the lnd server won't start until the chain height is non-zero, because it considers zero to be "not caught up."